### PR TITLE
enhance(api): add queue endpoint to return running and pending builds

### PR DESCRIPTION
**Pull request by JordanSussman**
Testing this locally returns:

```sh
$ curl -s -H 'Authorization: bearer <token>' http://localhost:8080/api/v1/admin/builds/queue?after=1614192207 | jq .
```

```json
[
  {
    "status": "running",
    "created": 1614194479,
    "number": 366,
    "full_name": "JordanSussman/vela-local-test"
  },
  {
    "status": "running",
    "created": 1614195579,
    "number": 367,
    "full_name": "JordanSussman/vela-local-test"
  },
  {
    "status": "running",
    "created": 1614195585,
    "number": 368,
    "full_name": "JordanSussman/vela-local-test"
  },
  {
    "status": "running",
    "created": 1614196363,
    "number": 371,
    "full_name": "JordanSussman/vela-local-test"
  },
  {
    "status": "running",
    "created": 1614196362,
    "number": 370,
    "full_name": "JordanSussman/vela-local-test"
  },
  {
    "status": "pending",
    "created": 1614196364,
    "number": 372,
    "full_name": "JordanSussman/vela-local-test"
  }
]
```

closes https://github.com/go-vela/community/issues/212

### CODE REVIEW

**Comment by wass3r** [comment]
thoughts on changing `timestamp` to `since`?

**Comment by JordanSussman** [comment]
> thoughts on changing `timestamp` to `since` or maybe `after`? slightly leaning more towards `after` for the potential to add a matching `before` to look at time slices of activity down the road?

Good idea! Swapped from `timestamp` to `after` to hopefully clarify usage.

File: api-spec/vela-server.json [hunk 1]
```diff
@@ -99,6 +99,48 @@
...
+        ],
+        "tags": [
+          "admin"
+        ],
+        "operationId": "AllBuildsQueue",
+        "parameters": [
+          {
+            "type": "string",
+            "description": "Unix timestamp to limit builds returned",
+            "name": "timestamp",
```
Comment (wass3r) [inline api-spec/vela-server.json:121]: might have to regenerate the spec

Comment (JordanSussman) [reply api-spec/vela-server.json]: Done.

**Review by wass3r — DISMISSED** [review]

File: Makefile [hunk 2]
```diff
@@ -249,3 +249,9 @@ compose-down:
 	@echo
 	@echo "### Destroying containers for docker-compose stack"
 	@docker-compose -f docker-compose.yml down
+
+.PHONY: swagger
+swagger:
```
Comment (wass3r) [inline Makefile:254]: thanks :D

File: api/admin/build.go [hunk 3]
```diff
@@ -57,6 +59,54 @@ func AllBuilds(c *gin.Context) {
...
+//     schema:
+//       type: array
+//       items:
+//         "$ref": "#/definitions/BuildQueue"
+//   '500':
+//     description: Unable to retrieve all running and pending builds from the database
+//     schema:
+//       "$ref": "#/definitions/Error"
+
+// AllQueue represents the API handler to
```
Comment (jbrockopp) [inline api/admin/build.go:89]: Could we update the comment here to reflect the function name?

```suggestion
// AllBuildsQueue represents the API handler to
```

Comment (JordanSussman) [reply api/admin/build.go]: Fixed.

**Review by jbrockopp — APPROVED** [review]
LGTM

**Review by wass3r — APPROVED** [review]

**Review by jbrockopp — COMMENTED** [review]
After some discussions with the team, it sounds like we're going to hold off on merging this.

We're currently in the process of finalizing some things on our end.

We'll be sure to merge this as soon as possible 👍

**Pull request merged**